filebeat/filebeat.reference.yml

```diff
@@ -388,6 +388,16 @@ filebeat.modules:
     # Filebeat will choose the the default path.
     #var.paths:
 
+#------------------------------ Shorewall Module ------------------------------
+- module: shorewall
+  # All logs
+  {fileset}:
```

Comment:
Please, leave this field just as `log`, which is the name of the only Fileset on the module.

Reply:
The thing is: I change that to `log` and the command `mage fmt update` changes it back to `{fileset}`

Reply:
Ok, some unknown magic only in mage. Forget about it and sorry for the comment :smile: